### PR TITLE
Provide version on API reference page for SDK and API

### LINKED ISSUE #84: Display current hyp3_sdk version number in API reference
The `hyp3_sdk` API reference hosted at https://hyp3-docs.asf.alaska.edu/using/sdk_api/ should include the current hyp3_sdk version number in the page title (or nearby the page title).

### PR DESCRIPTION
**Pull request by jhkennedy**
Fixes #84

### CODE REVIEW

File: docs/tools/asf_tools_api.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# `asf_tools` API Reference
+# `asf_tools` v0.2.0 API Reference
```
Comment (asjohnston-asf) [inline docs/tools/asf_tools_api.md:1]: Should we code-format the version number?
```suggestion
# `asf_tools v0.2.0` API Reference
```

File: docs/using/sdk_api.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
-# `hyp3_sdk` API Reference
+# `hyp3_sdk` v1.2.0 API Reference
```
Comment (asjohnston-asf) [inline docs/using/sdk_api.md:1]:
```suggestion
# `hyp3_sdk v1.2.0` API Reference
```

File: docs/tools/asf_tools_api.md (hunk 1 above)
Comment (asjohnston-asf) [reply docs/tools/asf_tools_api.md:1]: this means a change to the `sed` commands, however

Comment (jhkennedy) [reply docs/tools/asf_tools_api.md:1]: image
vs
image

I lean towards not code-formatting the version number but I don't feel strongly about it

Comment (Jlrine2) [reply docs/tools/asf_tools_api.md:1]: can we bold the "API Reference"

I think I'd prefer all 3 to look slightly different

Comment (jhkennedy) [reply docs/tools/asf_tools_api.md:1]: Hmm, this:
image
or
image

Comment (jhkennedy) [reply docs/tools/asf_tools_api.md]: Bottom one it is!

**Comment by jhkennedy** [comment]
I also added a `.gitignore` for Python, JetBrains, and VIM now that we have a python macro

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**